### PR TITLE
System contract new style

### PR DESCRIPTION
**Pull request by moskvanaft**

### CODE REVIEW

File: contracts/eosio.system/delegate_bandwidth.cpp [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+      account_name  from;
+      account_name  to;
+      asset         net_weight;
+      asset         cpu_weight;
+      asset         storage_stake;
+      uint64_t      storage_bytes = 0;
+
+      uint64_t  primary_key()const { return to; }
+
+      EOSLIB_SERIALIZE( delegated_bandwidth, (from)(to)(net_weight)(cpu_weight)(storage_stake)(storage_bytes) )
```
Comment (bytemaster) [inline contracts/eosio.system/delegate_bandwidth.cpp]: Is this required or does it work without?

File: contracts/eosio.system/delegate_bandwidth.cpp [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+   struct total_resources {
+      account_name  owner;
+      asset         net_weight;
+      asset         cpu_weight;
+      asset         storage_stake;   
+      uint64_t      storage_bytes = 0;
+
+      uint64_t primary_key()const { return owner; }
+
+      EOSLIB_SERIALIZE( total_resources, (owner)(net_weight)(cpu_weight)(storage_stake)(storage_bytes) )
```
Comment (bytemaster) [inline contracts/eosio.system/delegate_bandwidth.cpp]: Is this required or does it work without?

File: contracts/eosio.system/delegate_bandwidth.cpp [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+
+      EOSLIB_SERIALIZE( refund_request, (owner)(request_time)(amount) )
+   };
+
+   typedef eosio::multi_index< N(totalband), total_resources>   total_resources_table;
+   typedef eosio::multi_index< N(delband), delegated_bandwidth> del_bandwidth_table;
+   typedef eosio::multi_index< N(refunds), refund_request>      refunds_table;
+
+   void system_contract::delegatebw( const account_name from, const account_name receiver,
+                                            const asset stake_net_quantity, const asset stake_cpu_quantity,
```
Comment (bytemaster) [inline contracts/eosio.system/delegate_bandwidth.cpp]: align parameters with first row...

File: contracts/eosio.system/eosio.system.hpp [hunk 4]
```diff
@@ -26,176 +34,105 @@ namespace eosiosystem {
...
+                               >  producers_table;
+
+   typedef eosio::singleton<N(global), eosio_global_state> global_state_singleton;
+
+   static constexpr uint32_t     max_inflation_rate = 5;  // 5% annual inflation
+   static constexpr uint32_t     seconds_per_day = 24 * 3600;
+   static constexpr uint64_t     system_token_symbol = S(4,EOS);
+
+   class system_contract : public native, private eosio::contract {
+   public:
```
Comment (bytemaster) [inline contracts/eosio.system/eosio.system.hpp]: indent public

File: contracts/eosio.system/eosio.system.hpp [hunk 5]
```diff
@@ -26,176 +34,105 @@ namespace eosiosystem {
...
+
+      void increase_voting_power( account_name acnt, const eosio::asset& amount );
+
+      void decrease_voting_power( account_name acnt, const eosio::asset& amount );
+
+      static eosio_global_state get_default_parameters();
+
+      eosio::asset payment_per_block(uint32_t percent_of_max_inflation_rate);
+
+      void update_elected_producers(time cycle_time);
```
Comment (bytemaster) [inline contracts/eosio.system/eosio.system.hpp]: methods that are not "public" or "actions" should not be listed here.

File: contracts/eosio.system/delegate_bandwidth.cpp (hunk 2 above)
Comment (brianjohnson5972) [reply contracts/eosio.system/delegate_bandwidth.cpp]: How would the multi_index tables below use if these were not defined?

**Pull request merged**